public/content/wrapped-ether/index.md

```diff
@@ -0,0 +1,59 @@
+---
```

Comment:
should page slug be wrapped-ether or wrapped-eth?

Reply:
Will change to wrapped-eth after pulling in change requests